packages/fxa-shared/metrics/glean/fxa-backend-metrics.yaml

```diff
@@ -254,9 +282,370 @@ utm:
       . (period), % (percentage sign), and - (hyphen) in the allowed set of
       characters.
     type: string
-    lifetime: ping
+    lifetime: application
     send_in_pings:
       - accounts-events
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+
+access_token:
+  checked:
+    type: event
+    description: |
+      TODO
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+  created:
+    type: event
+    description: |
+      TODO
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7265
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+    extra_keys:
+      reason:
+        description: additional context-dependent info, e.g. the cause of an error
+        type: string
+login:
+  backup_code_success:
+    type: event
+    description: |
+      Login Backup Code Success (BE)
```

Comment:
BE = backend?

Reply:
Yes, I think so. I pulled all of these from the spreadsheet but this seems redundant here, will remove.